Protein/examples/0.11-RELEASE/nextprot-jsonld.json

```diff
@@ -0,0 +1,36 @@
+{
+    "@context": "http://schema.org",
+    "@type": "DataRecord",
+    "@id": "https://www.nextprot.org/entry/NX_P52701",
+    "includedInDataset": "ftp://ftp.nextprot.org/pub/current_release/xml/nextprot_all.xml.gz",
+    "citation": {
+      "@id": "",
+      "@type": ""
+    },
+    "mainEntity": {
+      "@id": "https://www.nextprot.org/entry/NX_P52701",
+      "@type": "Protein",
+      "http://purl.org/dc/terms/conformsTo": "https://bioschemas.org/specifications/Protein/0.9-DRAFT",
+      "identifier": "NX_P52701",
+      "name": "DNA mismatch repair protein Msh6",
+      "description": "",
```

Comment:
Omit properties for which there is no data. However in this case you probably want to include the text in your overview section of the webpage